tools/ci_build/github/mac-ios-swift-packaging-pipeline.yml

```diff
@@ -0,0 +1,117 @@
+parameters:
+- name: BuildType
+  displayName: |-
+    Type of build.
+    "normal": A normal build not for publication.
+  type: string      #TODO: add other build types here
+  values:
+  - normal
+  default: normal
+
+name: "$(Date:yyyyMMdd)$(Rev:rrr)"  # build number format
+
+stages:
+- stage: SPM_IOS_Local_Packaging_Testing
+  dependsOn: []
+  jobs:
+  - job: SPMIosPackaging
+    displayName: "SPM iOS Packaging Local"
+
+    pool:
+      vmImage: "macOS-13"
+
+    variables:
+      xcodeVersion: "14.3"
+      artifactsName: "ios_packaging_artifacts" #TODO: Add `_full` suffix when syncing to latest main
+
+    timeoutInMinutes: 300
+
+    steps:
+    - template: templates/use-xcode-version.yml
+      parameters:
+        xcodeVersion: ${{ variables.xcodeVersion }}
+
+    # Download artifacts from a specific pipeline 
+    # For now, it consumes rel-1.15.0 version ORT iOS Pod which matches the current source code
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'specific'
+        project: 'Lotus'
+        definition: 995
```

Comment:
Add comment saying how `definition: 995` was determined and what the ORT CI name is so that if the ORT iOS CI changes/goes away we know how to update this CI.